### PR TITLE
Update docs with wrap-instance command

### PR DESCRIPTION
**Pull request by krishnanrs**
Added docs with help and usage information on the wrap-instance
command

### CODE REVIEW

File: aws.md [hunk 1]
```diff
@@ -290,7 +322,7 @@ usage: brkt aws wrap-guest-image [-h] [--wrapped-instance-name NAME]
                                  [--token TOKEN | --brkt-tag NAME=VALUE]
                                  ID
 
-Launch guest image wrapped with Bracket Metavso
+Launch guest image wrapped with Bracket Metavsor
```
Comment (bburtin) [inline aws.md]: Is this a typo in the usage code?

Comment (krishnanrs) [reply aws.md]: No. This was a typo in the markdown doc. I identified this when updating it for the new (wrap-instance) command.

Comment (bburtin) [reply aws.md]: Ok, but there's still a typo in the correction.  :-)

Comment (krishnanrs) [reply aws.md]: Sorry, should have noticed that earlier!

**Pull request merged**